Program/Seasons.py

```diff
@@ -0,0 +1,220 @@
+import time
+
+
+
+#This is the stored pictures, calling "global" so the entire program can use them.
+setMediaPath("F:\COMP 120 Tinkering Graphics-JH182233\Pictures") 
+picfour = makePicture("Tree-Four-Seasons.jpg") 
+picwinter = makePicture("Tree-Winter.jpg") 
+picspring = makePicture("Tree-Spring.jpg") 
+picsummer = makePicture("Tree-Summer.jpg")
+picautumn = makePicture("Tree-Autumn.jpg")  
+picM = makeEmptyPicture(513, 556, black)
+pic = makeEmptyPicture(1025, 1110, black)
+
+
+#The main function for the program
+def main():
+#create an empty picture and populate it with others 
```

Comment:
The code in here is quite repetitive, perhaps you could make it into a function that takes the picture and start position as arguments?